scripts/start

```diff
@@ -1,23 +1,27 @@
 #!/bin/bash
 
-cd $(dirname $0)/..
```

Comment:
this was important - why did you remove the code to set proper path in case scripts/starts was called from a ddiferent directory?